edx_proctoring/templates/proctored_exam/submitted.html

```diff
@@ -7,6 +7,11 @@
   </h3>
   <p>
     {% blocktrans %}
+    If your proctoring client is still open, please close it now.
```

Comment:
This is more a question for @catong, but we may want to use "proctoring session" consistently.